src/main/java/seedu/address/logic/commands/PresentAttendanceCommand.java

```diff
@@ -0,0 +1,176 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DROPOFF;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PICKUP;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PETS;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import java.util.Optional;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.AttendanceUtil;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.attendance.PresentAttendanceEntry;
+import seedu.address.model.pet.AttendanceHashMap;
+import seedu.address.model.pet.Pet;
+
+/**
+ * Marks the attendance of an existing pet in WoofAreYou as present on a particular date.
+ */
+public class PresentAttendanceCommand extends Command {
```

Comment:
Just a thought, perhaps consider making an AttendanceCommand interface and have PresentAttendanceCommand and AbsentAttendanceCommand implement it? This may reduce code reusability as PresentAttendanceCommand and AbsentAttendanceCommand are really similar.

Reply:
I think I will leave it as it is for now, since present and absent commands require a different number of fields!

Reply:
Oh right, I see!